pkg/columns/examples/gadget/gadget.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
+	"github.com/kinvolk/inspektor-gadget/pkg/columns/formatter/textcolumns"
+)
+
+type Kubernetes struct {
+	Node      string `column:"node" columnTags:"kubernetes"`
+	Container string `column:"container" columnTags:"kubernetes"`
+	Pod       string `column:"pod" columnTags:"kubernetes"`
+}
+
+type Runtime struct {
+	Runtime string `column:"runtime" columnTags:"runtime"`
+}
+
+type GadgetData struct {
+	Kubernetes
+	Runtime
+	GadgetData string `column:"gadgetData"`
+}
+
+var GadgetOutput = []*GadgetData{
+	{
+		Kubernetes: Kubernetes{
+			Node:      "Node 1",
```

Comment:
Haven't thought about doing that in tests, good idea - however, as this is just an example, I think it might confuse more than help when creating the entries somewhere hidden in the code at runtime. If you worry about the line count, maybe we should just remove the "Node 3" entry.